src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.cpp

```diff
@@ -514,3 +514,20 @@ Value getDynamicMemRefSizeInBytes(
 
   return result;
 }
+
```

Comment:
Just add some comments here.

Consider the following "alloc(p0, p1, p2) : memref<1x?x2x?x?x3xf32, map>" with 3 parameters (in general pn parameters).

We are looking for the parameter p0..pn that correspond to the "?" dynamic parameter pointed by index in the memref. To find the right parameter, scan the memref to find the number of '?` that precede the one pointed to by index. Return that parameter index number

Reply:
Add in header file.